django/views/generic/base.py

```diff
@@ -71,12 +70,16 @@ def view(request, *args, **kwargs):
         view.view_class = cls
         view.view_initkwargs = initkwargs
 
-        # take name and docstring from class
-        update_wrapper(view, cls, updated=())
+        # Take special attributes as defined on the class, and annotations and
```

Comment:
I don't think we need comments about the attributes copied from the class, or about the annotations. Now we're not using `update_wrapper` this much is obvious from the code.

The comments about `__name__` and `__qualname__` are useful to have though. The explanation as to why we are copying any possible attributes set by decorators on dispatch is also very useful.

Reply:
Alright. Making the updates!